stories/covid-vax/covid_vax.ipynb

```diff
@@ -0,0 +1,1527 @@
+{
```

Comment:
<p>Two narratives. I propose to break the paragraph at "historical data first." The next sentence ("Let's take a look at the first row of our data:") can go in a new line with a technical color. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/138/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>There is a slight inconsistency on section headers, in terms of capitalisation and the use of questions. I like the two headers that follow, so wonder if that should be a model for this and the ones above?</p>